### DISCUSSION TITLE
Security Fixes - List of Changes?

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#15810] [How to] [answered]
Does there exist a (human readable) list of the changes made in each of the offered security fixes, like the one (YetiForce CRM 6.3 SecurityFix v8) pushed today?

**Comment by rskrzypczak** [comment, marked as answer]
The last patch was made because of this: CVE-2022-24775
As for the previous ones, they are mainly csrf-related bugs and other minor logical errors in the system.
We will try to specify the actions of individual patches in the future
